Framework/script/RepoCleaner/qcrepocleaner/o2-qc-repo-update-run-type

```diff
@@ -0,0 +1,107 @@
+#!/usr/bin/env python3
+
+import logging
+import argparse
+import dryable
+from qcrepocleaner.Ccdb import Ccdb
+from qcrepocleaner.binUtils import prepare_main_logger
+import csv
+
+
+def parse_args():
+    """Parse the arguments passed to the script."""
+    logging.info("Parsing arguments")
+    parser = argparse.ArgumentParser(description='Update the run type of all versions in a path. The run types for each run are '
+                                                 'provided in a csv file whose format is the one from the bookkeeping, with headers.')
+    parser.add_argument('--url', dest='url', action='store', help='URL of the CCDB, with http[s]://', required=True)
+    parser.add_argument('--log-level', dest='log_level', action='store', default="20",
+                        help='Log level (CRITICAL->50, ERROR->40, WARNING->30, INFO->20,DEBUG->10)')
+    parser.add_argument('--dry-run', action='store_true',
```

Comment:
is the `dry-run` actually respected? i don't see anything skipped because we recognize that we do a dry run.

Reply:
We do `    dryable.set(args.dry_run)`. 
Then in `ccdb.py` we have e.g.
```
    @dryable.Dryable()
    def deleteVersion(self, version: ObjectVersion):
```
Whenever `deleteVersion` is called, it will be skipped and a message printed if dryable was turned on.

Reply:
ok, clear, thanks